app/models/piece.rb

```diff
@@ -15,6 +15,14 @@ def move_to!(x,y)
     end
   end
 
+  def y_values(y)
```

Comment:
@Brandongoodman615 I really like the refactoring. One really small addition would be to give the `y_values` and `x_values` a bit more descriptive names.. maybe sth like `y_offset` 😄